src/main/java/hudson/plugins/im/bot/HealthCommand.java

```diff
@@ -26,17 +26,22 @@
     @Override
     protected CharSequence getMessageForJob(AbstractProject<?, ?> project) {
         StringBuilder msg = new StringBuilder(32);
+        AbstractBuild<?, ?> lastBuild = project.getLastBuild();
         msg.append(project.getFullDisplayName());
         if (project.isDisabled()) {
             msg.append("(disabled)");
         } else if (project.isBuilding()) {
-            msg.append("(BUILDING: ").append(project.getLastBuild().getDurationString()).append(")");
+            msg.append("(BUILDING");
+            if (lastBuild != null) {
+                msg.append(": ");
```

Comment:
Aren't mixing things here? Though this change does seem to make sense, is it related to supporting display-url-plugin?